### DISCUSSION TITLE
Obligatory project check in. Is V7-alpha still alive?

### DISCUSSION THREAD

**Discussion by mpkamowski** [sequelize/sequelize#17451] [Help & Questions] [answered]
Noticed a huge drop off in all activity in the project and just want to make sure things are ok before we start looking at alternatives. I understand people get busy and thats fine but just want check in.

Also, thank you for the work you've done! I respect and appreciate all that you have contributed!

**Comment by WikiRik** [comment, marked as answer]
Hi! It's indeed been busy for the maintainers in the last months so there's been very limited progress. I hope to dedicate some more time again in about a month, but no promises. Thanks for checking in!
